frontend/app/(dashboard)/layout.tsx

```
@@ -17,19 +17,41 @@ const routeTitles: { path: string; title: string; exact?: boolean }[] = [
-      return localStorage.getItem('nimbus-sidebar-collapsed') === 'true'
-    }
-    return false
-  })
+  const isDesktopCollapsed = useSyncExternalStore(
+    subscribeSidebar,
+    getSidebarSnapshot,
+    getSidebarServerSnapshot
+  )
+  const setIsDesktopCollapsed = useCallback((value: boolean) => {
+    localStorage.setItem('nimbus-sidebar-collapsed', String(value))
+    sidebarListeners.forEach((l) => l())
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)